djangocms_icon/templates/admin/djangocms_icon/change_form.html

```diff
@@ -0,0 +1,9 @@
+{% extends "admin/cms/page/plugin/change_form.html" %}
+{% load staticfiles %}
+
+{% block extrahead %}
+    {{ block.super }}
+    {# here go the iconset css files #}
+
+    <link rel="stylesheet" href="{% static 'djangocms_icon/css/font-awesome.min.css' %}" media="all">
```

Comment:
maybe we should use an include here instead, like "djangocms_icon/templates/djangocms_icon/custom_icons.html"

Reply:
I'd rather not have them change the admin template on a project basis.

Reply:
i disagree. this _is_ an admin template for a reason
performance aside, i'd rather not introduce any more templates than we need

Reply:
okay :(